### DISCUSSION TITLE
Add favorite in navigation / follow

### DISCUSSION THREAD

**Discussion by bios4** [devemux86/cruiser#323] [Q&A] [answered]
_Added as a new thread as requested by @devemux86 [here](https://github.com/devemux86/cruiser/discussions/296#discussioncomment-7079829)_ 

Suggestion: provide a new button on screen during navigation / follow which immediately saves the current location (GPS coordinates, street name, ...) as a "temporary favorite" every time it is pressed.

Similar to the "highlight tag" in modern GoPro cameras, which add sort of a "bookmark" within a video recording as soon as the corresponding button is pressed.

These temporary favorites can then be updated (with proper name, favorite group etc.) after the trip.

The button should be configurable so that it is only shown for users which use this feature. ;-)

**Comment by devemux86** [comment]
@bios4 thanks for the idea.

Something like this is indeed useful, it is already in my list for implementation. :slightly_smiling_face:

**Comment by DolomitesTom** [comment]
This is something I had already proposed with "Wow" button. I thought it was already on the list.

**Reply by devemux86** [reply]
@DolomitesTom it was already indeed with a different description from our last discussion. :slightly_smiling_face:

**Comment by devemux86** [comment, marked as answer]
Let's discuss all quick actions during navigation / follow in:
- https://github.com/devemux86/cruiser/discussions/405
